symphony/app/fbcnms-projects/magmalte/app/components/DashboardAlertTable.js

```diff
@@ -10,17 +10,30 @@
 
 import LoadingFiller from '@fbcnms/ui/components/LoadingFiller';
 import MagmaV1API from '@fbcnms/magma-api/client/WebClient';
-import Paper from '@material-ui/core/Paper';
+import Card from '@material-ui/core/Card';
 import React from 'react';
 import TabbedTable from './TabbedTable';
-import Text from '@fbcnms/ui/components/design-system/Text';
+import Text from '../theme/design-system/Text';
+import {colors} from '../theme/default';
 import nullthrows from '@fbcnms/util/nullthrows';
 import useMagmaAPI from '@fbcnms/ui/magma/useMagmaAPI';
+import Grid from '@material-ui/core/Grid';
 import {Alarm} from '@material-ui/icons';
+import {makeStyles} from '@material-ui/styles';
 import {useRouter} from '@fbcnms/ui/hooks';
 import type {RowData} from './TabbedTable';
```

Comment:
nit: let's make sure this is ordered alphabetically, and we usually put an empty line between module vs function imports